<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Function3dDialog</class>
 <widget class="QDialog" name="Function3dDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Funkcja 3d</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>f(x,y) = </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fstring"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Kolor wykresu:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="ColorButton" name="colorBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Przezroczystość:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="alphaSlider">
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Jakość kreślenia: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="qualitySlider">
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Metoda kreślenia</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="quadsRadioBtn">
        <property name="text">
         <string>czworokąty (GL_QUADS)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="trianglesRadioBtn">
        <property name="text">
         <string>trójkąty (GL_TRIANGLES)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="linesRadioBtn">
        <property name="text">
         <string>linie (GL_LINES)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pointsRadioBtn">
        <property name="text">
         <string>punkty (GL_POINTS)</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>quadsRadioBtn</zorder>
     <zorder>linesRadioBtn</zorder>
     <zorder>pointsRadioBtn</zorder>
     <zorder>trianglesRadioBtn</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="drawInIntervalGroupBox">
     <property name="title">
      <string>Kreśl w przedziale</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Od (x min):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="intervalXFrom">
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Do (x max):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="intervalXTo">
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Od (y min):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="intervalYFrom">
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Do (y max)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="intervalYTo">
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fstring</tabstop>
  <tabstop>colorBtn</tabstop>
  <tabstop>alphaSlider</tabstop>
  <tabstop>qualitySlider</tabstop>
  <tabstop>quadsRadioBtn</tabstop>
  <tabstop>trianglesRadioBtn</tabstop>
  <tabstop>linesRadioBtn</tabstop>
  <tabstop>pointsRadioBtn</tabstop>
  <tabstop>drawInIntervalGroupBox</tabstop>
  <tabstop>intervalXFrom</tabstop>
  <tabstop>intervalXTo</tabstop>
  <tabstop>intervalYFrom</tabstop>
  <tabstop>intervalYTo</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>Function3dDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
